### PR TITLE
Ignore unused typedef in _GPBCompileAssert macro

### PR DESCRIPTION
**Pull request by MaxGabriel**
This fixes a warning about an unused typedef:

```
Ignore unused typedef in _GPBCompileAssert macro
```

I got this error when adding the Protobuf code to my project, but I can't figure out how to trigger the warning in the Protobuf project. I'm using Xcode 7.0 in both projects.
#285 is kind of hard to parse, but this may fix the compiler error reported in this.

### CODE REVIEW

**Comment by MaxGabriel** [comment]
I signed it!

**Comment by haberman** [comment]
It looks like this specific PR is obsolete as the given symbol `_GPBCompileAssertSymbol` no longer exists.
